qiskit/assembler/assemble_schedules.py

```diff
@@ -75,6 +76,14 @@ def assemble_schedules(schedules, qobj_id, qobj_header, run_config):
         for shift, instruction in schedule.instructions:
             # TODO: support conditional gate
 
+            if isinstance(instruction, ParametricInstruction):
+                pulse_shape = ParametricPulseShapes(type(instruction.command)).name
+                if pulse_shape not in run_config.parametric_pulses:
+                    # Convert to SamplePulse if the backend does not support it
+                    instruction = PulseInstruction(instruction.command.get_sample_pulse(),
```

Comment:
This might be a corner case, but what happens if there are two identical pulses in a schedule and one is defined by ParametricPulse and other is defined by discrete pulse? they can have the same name.

Reply:
`parametric_pulse` is the only new special keyword -- so discrete pulses can't have the name `parametric_pulse`, just like it can't be called `fc`, `pv`, etc

Reply:
no, I'm asking what happens on the backend side. Currently different pulses with the identical name is detected when pulse library is generated and name of second pulse is modified not to overwrap. the same mechanism exists on the backend?

Reply:
Having a gaussian parametric and gaussian sample will work on the backend. Or having many of the same parametric or many samples w/ same name (for samples, I believe name needs to be anonymized to do this)

Reply:
I see. the problem here is `ParametricPulse` can slip through the name duplication check. Duplicated names only matters at transport layer?

Reply:
We should probably keep discussion about our backend code to internal channels